src/yb/rocksdb/table_properties.h

```diff
@@ -76,6 +76,8 @@ struct TableProperties {
   // The name of the filter policy used in this table.
   // If no filter policy is used, `filter_policy_name` will be an empty string.
   std::string filter_policy_name;
+  // Cache key prefix used for detecting collision of SST table.
+  std::string cache_key_prefix = "";
```

Comment:
`metadata_cache_key_prefix_hex_str`? 
BTW, do we need data file cache key prefix as well?